swap/src/config.rs

```diff
@@ -1,5 +1,4 @@
 pub mod seed;
-
 use crate::bitcoin::Timelock;
```

Comment:
A line separating the `pub mod` and `use` is good as they are different items.

Reply:
Unfortunately it's not that simple... I played around a bit and keeping the newline between `use` and `pub use` imports can lead to the same problem we had with `use::crate`.

If we have `use` and `use::crate` \n` `pub use` then newly added `use` statements might be added to the `pub use` section by Clion.

That can be overcome by putting all `pub use` statements on the top. Would that be acceptable? 

Note: The problem only occurs with multiple `pub use` statements, I tried some auto importing in `bitcoin.rs` to explore the behavior.